bylaws.md

```diff
@@ -0,0 +1,602 @@
+<center>
+#Bylaws
+
+#of
+
+#DJANGO EVENTS FOUNDATION NORTH AMERICA
+
+#A California Public Benefit Corporation
+</center>
+
+## ARTICLE 1 - OFFICES ##
+
+### SECTION 1. PRINCIPAL OFFICE ###
+
+The principal office of the corporation for the transaction of its business is located in Santa Clara County, California.
+
+### SECTION 2. CHANGE OF ADDRESS ###
+
+The county of the corporation’s principal office can be changed only by amendment of these bylaws and not otherwise. The board of directors may, however, change the principal office from one location to another within the named county by noting the changed address and effective date below, and such changes of address shall not be deemed an amendment of these bylaws.
+
+__________________	Dated: __________________
+
+__________________	Dated: __________________
+
+__________________	Dated: __________________
+
+### SECTION 3. OTHER OFFICES ###
+
+The corporation may also have offices at such other places, within or without the State of California, where it is qualified to do business, as its business may require and as the board of directors may, from time to time, designate.
+
+## ARTICLE 2 - PURPOSES ##
+
+### SECTION 1. OBJECTIVES AND PURPOSES ###
+
+The primary objectives and purposes of this corporation shall be: the provision of educational events for users of the Django software application.
+
+## ARTICLE 3 - DIRECTORS ##
+
+### SECTION 1. NUMBER ###
+
+The corporation shall have a minimum of three directors, and a maximum of nine directors, and collectively they shall be known as the board of directors. The number may be changed by amendment of this bylaw, or by repeal of this bylaw and adoption of a new bylaw, as provided in these bylaws.
+
+### SECTION 2. POWERS ###
+
+Subject to the provisions of the California Nonprofit Public Benefit Corporation law and any limitations in the articles of incorporation and bylaws relating to action required or permitted to be taken or approved by the members, if any, of this corporation, the activities and affairs of this corporation shall be conducted and all corporate powers shall be exercised by or under the direction of the board of directors.
+
+### SECTION 3. DUTIES ###
+
+It shall be the duty of the directors to:
+
+(a) Perform any and all duties imposed on them collectively or individually by law, by the articles of incorporation of this corporation, or by these bylaws;
+
+(b) Appoint and remove, employ and discharge, and, except as otherwise provided in these bylaws, prescribe the duties and fix the compensation, if any, of all officers, agents, and employees of the corporation;
+
+(c) Supervise all officers, agents, and employees of the corporation to assure that their duties are performed properly;
+
+(d) Meet at such times and places as required by these bylaws;
+
+(e) Register their addresses with the secretary of the corporation and notices of meetings mailed or telegraphed to them at such addresses shall be valid notices thereof.
```

Comment:
Should we convert these to markdown lists?

Reply:
Let's follow up on this if you want to create a new issue. Markdown tends to auto-renumber which we probably don't want for finding stuff quickly, but maybe I'm missing how you want to structure it.

Reply:
No, this is totally fine. It's not worth changing if you're good with the manual list.